### ISSUE TITLE
[Feature request] Add option for "inbox" to take you to `/notifications` instead of `/message/unread`

### ISSUE THREAD

**Issue by NSExceptional** [voidref/orangered#11]
Looks like Reddit is deprecating `/message/unread` for some users on New Reddit, and instead shows notifications over at `/notifications` now

**Comment by voidref** [comment]
Is this happening for you?

I just updated the code ... a bunch, and it's not even letting me log in any more.

Check it out:

https://github.com/voidref/orangered/tree/bit-rot-work

Does this let you log in?

**Comment by voidref** [comment]
Looks like OAuth needs to be implemented ... not sure I'm really motivated for that kind of effort.

PRs welcome.

**Comment by NSExceptional** [comment]
Oof, me either. Scared to try a new version because of that. I might just hex edit the binary to open the new address instead. I'll make a new issue about OAuth and make this a dependent of it, if you want. We can label it Help Wanted and maybe someone will help out?

**Comment by voidref** [comment]
Sounds like a plan, make it so!

**Comment by tinder-tannerbennett** [comment]
Done, #13. I can't add the "help wanted" label though, I think you have to.

So, is OAuth really a prerequisite for new features? Does updating the app log you out or something? Or could this still be done with an update without OAuth?

**Comment by tinder-tannerbennett** [comment]
Heh, sorry, I'm on my work computer. I'm @NSExceptional

**Comment by voidref** [comment]
I don't think so, it should still work if you have the cached auth. You could always try just changing the path in the source and recompiling.
